src/middlewared/middlewared/plugins/rsync.py

```diff
@@ -343,3 +346,165 @@ class Config:
     @accepts(Int('id'))
     async def do_delete(self, id):
         return await self.middleware.call('datastore.delete', self._config.datastore, id)
+
+
+class RsyncTaskService(CRUDService):
+
+    class Config:
+        datastore = 'tasks.rsync'
+        datastore_prefix = 'rsync_'
+
+    async def validate_rsync_task(self, data):
+        verrors = ValidationErrors()
+
+        if ' ' in data.get("user"):
+            raise verrors.add("user", "Usernames cannot have spaces")
```

Comment:
Every validation error needs the schema name to not be ambiguous: e.g. rsync_task.user

Reply:
I suggest passing `schema_name` to `validate_rsync_task` as we do in other plugins